doc/settings.rst

```diff
@@ -109,7 +109,7 @@ You can load it with:
   import yaml
 
   with open('settings.yml') as config:
-       settings_dict = yaml.load(config)
+       settings_dict = yaml.full_load(config)
```

Comment:
Wouldn't safe_load make more sense here?

Reply:
Probably... I was just not sure, but if you think it's enough, I will change it.
I just don't know, what is the exact difference.

Reply:
`full_load` supports unsafe things that could lead to exploits, `safe_load` doesn't. In practice I haven't seen a case yet where `full_load` is necessary. In general it is recommended to always use `safe_load`, so I think we should use that as well.

Reply:
BTW the FullConstructor is still much safer then the legacy UnsafeConstructor.

Reply:
Ok I've changed it.